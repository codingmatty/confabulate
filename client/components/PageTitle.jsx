import UtilityText from './styled/UtilityText';

export default function PageTitle({ children }) {
  return <UtilityText as="h1">{children}</UtilityText>;
}
