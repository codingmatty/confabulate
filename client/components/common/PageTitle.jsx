import PropTypes from 'prop-types';
import UtilityText from './UtilityText';

export default function PageTitle({ children }) {
  return <UtilityText as="h1">{children}</UtilityText>;
}
PageTitle.propTypes = {
  children: PropTypes.node.isRequired
};
