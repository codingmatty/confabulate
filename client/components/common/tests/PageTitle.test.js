import { cleanup } from 'react-testing-library';
import renderWithTheme from '../../../test-utils/render-with-theme';
import PageTitle from '../PageTitle';

describe('PageTitle', () => {
  afterEach(cleanup);

  it('matches default snapshot', () => {
    const { container } = renderWithTheme(<PageTitle>Page Title</PageTitle>);
    expect(container).toMatchSnapshot();
  });

  it('renders h1', () => {
    const { container } = renderWithTheme(<PageTitle>Page Title</PageTitle>);
    expect(container.firstChild.tagName).toBe('H1');
  });
});
