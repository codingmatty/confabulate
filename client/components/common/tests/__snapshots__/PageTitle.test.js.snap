// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTitle matches default snapshot 1`] = `
.c0 {
  color: #7E7E7E;
  font-size: 1rem;
  margin: 0 0 1em;
  text-transform: uppercase;
}

<div>
  <h1
    class="c0"
  >
    Page Title
  </h1>
</div>
`;
